src/core_modules/capture-core/utils/routing/newDashboard.js

```diff
@@ -0,0 +1,2 @@
+export const shouldUseNewDashboard = (userDataStore, dataStore, programId) =>
```

Comment:
Hey,
Great idea to move this helper function to the `/utils` folder. Does it make sense to import and use it everywhere, like in [navigateToEnrollmentOverview](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/actions/navigateToEnrollmentOverview/navigateToEnrollmentOverview.epics.js#L37) and [RegistrationDataEntry](https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/Pages/New/RegistrationDataEntry/RegistrationDataEntry.epics.js#L42)?
Thanks!

Reply:
Makes sense! I refactored the code since @jasminenguyennn is off today (and we will make an effort to get this included in the 2.39.0 bundled version)